plugins/Denizen/scripts/movement_skillz/double_jump.dsc

```diff
@@ -18,11 +22,9 @@ double_jump_handler:
         # How much time the player has to perform the double jump before it's too late
         time_to_jump: 0.5s
         # This basically indicates how far you'll go with the double jump
-        vector_multiplicator: 2
-        # Chooses whether to use .forward_flat or .direction.vector. One launches you straight forward while the other takes in user vertical input,
-        # meaning you could double jump directly upwards if you looked to the sky.
-        ## WILL STILL TAKE FALL DAMAGE!
-        verticality: false
+        vector_multiplier: 2
+        # multiplier of the vector in case of verticality. Meant to prevent it from being too OP.
+        verticality_multiplier: 0.7
```

Comment:
The comment mentioned it would only use half of the vector, but this is 70% of it. Can you clarify?